src/main/scala/com/microsoft/hyperspace/HyperspaceException.scala

```diff
@@ -17,3 +17,5 @@
 package com.microsoft.hyperspace
 
 case class HyperspaceException(msg: String) extends Exception(msg)
+
+case class NoChangesDetected(msg: String) extends Exception(msg)
```

Comment:
`NoChangeException` (and with some comments when this will be used)?

Reply:
Also can you scope this to `private[actions]`?

Reply:
thanks @imback82 , fixed